themes/vald/layouts/partials/sidebar.html

```diff
@@ -2,7 +2,7 @@
 <!-- TODO: attach class "open" nav -->
 <aside class="page">
   {{ $current := .Page.Permalink }}
-  <nav class="open">
+  <nav>
```

Comment:
```suggestion
  <nav class="open">
```

we should keep attach `open` class before implementation.

Reply:
I changed here in this commit
https://github.com/vdaas/web/pull/21/commits/057e001a95ad796a5c1c5eb104fc8c6d2d5f0cbe